src/WatchlistCard.css

```diff
@@ -1 +1,9 @@
-WatchlistCard.css
\ No newline at end of file
+.watchlist-card {
+  height: 110px;
```

Comment:
This is the perfect size for out watchlist cards. 
Next step: let's add a background image for the episodes in our watchlists.